src/main/java/mate/academy/config/SecurityConfig.java

```diff
@@ -37,7 +37,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(
                         auth -> auth
-                                .requestMatchers("/auth/**", "/swagger-ui/**")
+                                .requestMatchers("/auth/**", "/swagger-ui/**", "/categories/**")
```

Comment:
why we should allow full access to the `/categories/**` endpoints?
```suggestion
                                .requestMatchers("/auth/**", "/swagger-ui/**")
```